workshop/lab1.3_configure.md

```diff
@@ -0,0 +1,123 @@
+# Setup
+
+As a preqrequisite to running these labs, you will need access to an OpenShift v4.x cluster as a cluster admin.  We also assume you have the `oc` client installed and have logged into your cluster with it.
+
+
+## Operator Installation
+
+In OpenShift, installation of Knative is very easy.  You just need to find and install the OpenShift Serverless Operator.
+
+![OpenShift Serverless Operator Hub](content/images/serverless_operator_hub.png)*Operator Hub*
+
+Then click install.
+
+![Install OpenShift Serverless Operator](content/images/serverless_install_operator.png)*OpenShift Serverless Operator*
+
+Next make sure to install in all namespaces.  Select the latest Update Channel made available to you.  For the purposes of this demo, we'll pick the Manual approval strategy.  Lastly, click Subscribe.
+
+![Subscribe to OpenShift Serverless Operator](content/images/serverless_create_subscription.png)*Subscribe*
+
+Wait a few minutes and eventually you will see it show up as Installed Operator with the status of Succeeded
+
+![Subscribe to OpenShift Serverless Operator](content/images/serverless_installed_operator.png)*Installed*
+
+
+## Knative Serving Installation
```

Comment:
any reason these are CLI and the operator is via webconsole? Seems a little odd - if we are doing it to showcase the flexibility to use webconsole or CLI, I think we should call that out with a quick little blurb:
"OpenShift Serverless install can be completely done with Operator Hub webconsole or the CLI - we are going to give you a little taste of both by adding the operator via webconsole and then the rest via CLI"

Reply:
In the future, we will need to pin our workshop to a specific version of Serverless.  So, we'll have to use the `ClusterServiceVersion` API.   I actually think we should do this now.

Reply:
Requesting to do this in another PR.  I opened an issue to track it https://github.com/RedHatGov/serverless-workshop-dashboard/issues/3.